components/menu/SubMenu.razor.cs

```diff
@@ -87,7 +87,7 @@ private void SetClass()
                         var eval = RootMenu?.InternalMode == MenuMode.Inline && IsOpen;
                         if (Key == "sub1")
                         {
-                            Console.WriteLine(eval);
+                            //Console.WriteLine(eval);
                         }
```

Comment:
Please delete this block.

Reply:
I also commented out some other calls which had some more text in them. Should I just remove all the calls also the ones that are commented out?

Reply:
@ElderJames updated.